otter/test/test_util.py

```diff
@@ -564,20 +574,27 @@ def test_method_failure(self):
 
         d = with_lock(self.reactor, self.lock, self.method, self.log)
         self.assertNoResult(d)
-        self.log.msg.assert_called_once_with('Starting lock acquisition')
+        self.log.msg.assert_called_once_with(
+            'Starting lock acquisition', lock_status='Acquiring',
+            **self.log_fields)
 
         self.reactor.advance(10)
         self.acquire_d.callback(None)
         self.assertEqual(
             self.log.msg.mock_calls[-2:],
-            [mock.call('Lock acquisition in 10.0 seconds', acquire_time=10.0),
-             mock.call('Starting lock release')])
+            [mock.call('Lock acquisition in 10.0 seconds',
+                       acquire_time=10.0, lock_status='Acquired',
+                       **self.log_fields),
+             mock.call('Starting lock release', lock_status='Releasing',
+                       **self.log_fields)])
         self.method.assert_called_once_with()
 
         self.reactor.advance(3)
         self.release_d.callback(None)
         self.log.msg.assert_called_with('Lock release in 3.0 seconds',
```

Comment:
Non-blocking - just to test that that message is not logged when the lock has not been held for that long (for the sake of test coverage, since the logic seems like it wouldn't do that), would it make sense to add a call to `self.reactor.advance(120)` here and maybe in a few places?